test_data/asm/vm_to_vm.asm

```diff
@@ -14,9 +14,8 @@ machine Main {
 
     function main {
         A <== add(1, 1);
-        // TODO: uncomment the following two lines once we support having many calls to a machine
-        // A <== add(A, 1);
-        // A <== sub(A, 1);
```

Comment:
This already worked before, right?

Reply:
Yes. The sigma logic that we removed in #690 prohibited VMs to be called more than once, that's why we had this TODO. There just wasn't a satisfying witness. But since it's gone now, we can actually call VMs as many times as we want.